config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  root "home#index"
+  get 'about', to: 'about#index'
```

Comment:
You can write this in the rails way
`resources :about, only: :index`

This can be applied to all routes